src/catalog/postgres/pg_proc_impl.cpp

```diff
@@ -78,14 +79,19 @@ std::vector<execution::functions::FunctionContext *> PgProcImpl::TearDownGetFunc
     }
   }
 
-  return func_contexts;
+  delete[] buffer;
+  return [func_contexts]() {
```

Comment:
nit: I think we want to use `std::move` here to avoid a deep copy of the vector.

Reply:
I forgot that existed. thanks!